filebeat/module/apache2/access/test/test.log-expected.json

```diff
@@ -0,0 +1,140 @@
+[
+{
+  "_index": "filebeat-2016.12.27",
+  "_type": "log",
+  "_id": "AVlBCaYsqYg9cc5KQfcT",
+  "_score": null,
+  "_source": {
+    "@timestamp": "2016-12-26T14:16:29.000Z",
+    "offset": 73,
+    "apache2": {
+      "access": {
+        "response_code": "404",
+        "remote_ip": "::1",
+        "method": "GET",
+        "user_name": "-",
+        "http_version": "1.1",
+        "body_sent": {
+          "bytes": "209"
+        },
+        "url": "/favicon.ico"
+      }
+    },
+    "beat": {
+      "hostname": "192-168-0-7.rdsnet.ro",
+      "name": "192-168-0-7.rdsnet.ro",
+      "version": "6.0.0-alpha1"
+    },
+    "input_type": "log",
+    "read_timestamp": "2016-12-27T15:52:23.304Z",
+    "source": "module/apache2/access/test/test.log",
+    "fields": {
+      "pipeline_id": "apache2-access-with_plugins",
+      "source_type": "apache2-access"
+    },
+    "type": "log"
+  },
+  "fields": {
+    "@timestamp": [
+      1482761789000
+    ]
+  },
+  "sort": [
+    1482761789000
+  ]
+},
+{
+  "_index": "filebeat-2016.12.27",
+  "_type": "log",
+  "_id": "AVlBCaYsqYg9cc5KQfcU",
+  "_score": null,
+  "_source": {
+    "@timestamp": "2016-12-26T16:22:13.000Z",
+    "offset": 238,
+    "apache2": {
+      "access": {
+        "referrer": "-",
+        "response_code": "404",
+        "remote_ip": "192.168.33.1",
+        "method": "GET",
+        "user_name": "-",
+        "http_version": "1.1",
+        "body_sent": {
+          "bytes": "499"
+        },
+        "url": "/hello",
+        "user_agent": {
+          "major": "50",
+          "minor": "0",
+          "os": "Mac OS X 10.12",
```

Comment:
It's what the processor does by default. I'd stick with that, since if we don't, someone will ask why it's different from LS.